packages/flutter_tools/lib/src/mdns_discovery.dart

```diff
@@ -57,7 +57,7 @@ class MDnsVmServiceDiscovery {
   /// The [deviceVmservicePort] parameter may be used to specify which port
   /// to find.
   ///
-  /// The [isNetworkDevice] parameter flags whether to get the device IP
+  /// The [useDeviceIPAsHost] parameter flags whether to get the device IP
```

Comment:
I read this as `use Device IPAs Host` (like an .ipa) which is confusing but I don't have a better suggestion.